installers/MacInstaller.hs

```diff
@@ -135,18 +107,18 @@ makeInstaller cfg = do
   de <- doesFileExist (dir </> "Frontend")
   unless de $ renameFile (dir </> "Daedalus") (dir </> "Frontend")
   run "chmod" ["+x", toText (dir </> "Frontend")]
-  writeLauncherFile dir cfg
+  writeLauncherFile dir
 
-  with (makeScriptsDir cfg) $ \scriptsDir -> do
+  with (makeScriptsDir opts) $ \scriptsDir -> do
     let
       pkgargs :: [ T.Text ]
       pkgargs =
            [ "--identifier"
-           , "org." <> appNameLowercase cfg <> ".pkg"
+           , "org.daedalus.pkg"
```

Comment:
This is a regression for mantis installer

Reply:
Done